beginner/webserver/index.html

```diff
@@ -1,31 +1,4 @@
 <!DOCTYPE html>
-
-<head>
-	<title>Docker Example</title>
-	<style>
```

Comment:
Can we add the styling again? Or why did you remove this?

Reply:
I agree with @the-technat. It would be nice to use the current styling.